rllib/algorithms/marwil/marwil.py

```diff
@@ -1,4 +1,4 @@
-from typing import Callable, Optional, Type, Union
```

Comment:
Clean up marwil. It was surfaced that the default of marwil was set to use IS and WIS by default which doesn't make sense.